src/components/hero.jsx

```
@@ -75,61 +75,61 @@
   };
 
   return (
-    <nav className='hero--container'> 
-      <div className='hero--banner--container'>
-        <img src={BannerPhoto} alt="Central Banner" className='hero--banner' onClick={refreshPage} />
+    <nav className='flex flex-col justify-center items-center bg-white shadow-lg rounded-2xl pb-5'>
+      <div className='transition-transform duration-300 ease-in-out cursor-pointer'>
+        <img src={BannerPhoto} alt="Central Banner" className='max-w-screen-lg mx-auto' onClick={refreshPage} />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)